### DISCUSSION TITLE
Swiper Grid doesn't work as expected in order.

### DISCUSSION THREAD

**Discussion by strongphoenix** [nolimits4web/swiper#7198] [Q&A]
I am using React.js, swiper to make slider.
Total items are 5 and I want to show 2 items in per page.
But it shows all items in the first page.
And second, third page is empty.

Expected result:
![image](https://github.com/nolimits4web/swiper/assets/80874362/f0ff8d27-4a12-4c14-808f-f8919875041e)

Current result:
![image](https://github.com/nolimits4web/swiper/assets/80874362/01a77e8e-74b2-4188-9629-0503860ee5de)

Here is my code.
```
<Swiper
            ref={swiperRef}
            spaceBetween={0}
            allowTouchMove={true}
            touchMove={true}
            grid={{
              // fill: 'row',
              rows: 2,
            }}
            pagination={{
              type: 'fraction'
            }}
            className="mySwiper"
            modules={[Grid, Pagination]}
            onInit={(swiper) => {
              setPagination(swiper?.pagination?.el?.outerText)
            }}
            onSlideChange={onChangeSlider}
          >
            {results?.map((item, index) => (
              <SwiperSlide key={index}>
                <div>{index + 1}</div>
              </SwiperSlide>
            ))}
          </Swiper>
```

How to fix this?
Swiper version: 11.0.4
React version: 18

**Comment by maryma00012** [comment]
have you tried to add the fill?
grid={{
    fill: 'row',
    rows: 2,
  }}
